### PR TITLE
NXJS-83: add collections related methods

### PR DESCRIPTION
**Pull request by troger**

### CODE REVIEW

File: lib/document.js [hunk 1]
```diff
@@ -82,6 +82,22 @@ class Document extends Base {
   }
 
   /**
+   * Returns weither this document is a collection or not.
```
Comment (ataillefer) [inline lib/document.js]: whether

File: lib/document.js [hunk 2]
```diff
@@ -82,6 +82,22 @@ class Document extends Base {
...
+   * @returns {Boolean} true if this document is folderish, false otherwise.
```
Comment (ataillefer) [inline lib/document.js]: if this document is a collection

**Comment by ataillefer** [comment]
:+1:

**Comment by kevinleturc** [comment]
👍

**Pull request merged**